docs/Architecture.md

```diff
@@ -1,49 +1,93 @@
-  * It receives the raw flows from the agent and decorates them with Kubernetes information (Pod
-    and host names, namespaces, etc.), and stores them as JSON into a [Loki](https://grafana.com/oss/loki/)
-    instance.
-* [Network Observability Console Plugin](https://github.com/netobserv/network-observability-console-plugin)
-  * It is attached to the Openshift console as a plugin (see Figure 1, though it can be also
-    deployed in standalone mode). The Console Plugin queries the flows information stored in Loki
-    and allows filtering flows, showing network topologies, etc.
-
-![Netobserv frontend architecture](./assets/frontend.png)
-Figure 1: Console Plugin deployment
-
-There are two existing deployment modes for Network Observability: direct mode and Kafka mode.
-
-## Direct-mode deployment
-
-In direct mode (figure 2), the eBPF agent sends the flows information to Flowlogs-Pipeline encoded as Protocol
-Buffers (binary representation) via [gRPC](https://grpc.io/). In this scenario, Flowlogs-Pipeline
-is usually deployed as a DaemonSet so there is a 1:1 communication between the Agent and FLP internal
-to the host, so we minimize cluster network usage.
-
-![Netobserv component's architecture (direct mode)](./assets/architecture-direct.png)
-Figure 2: Direct deployment
-
-## Kafka-mode deployment
-
-In Kafka mode (figure 3), the communication between the eBFP agent and FLP is done via a Kafka topic.
-
-![Netobserv component's architecture (Kafka mode)](./assets/architecture-kafka.png)
-Figure 3: Kafka deployment
-
-This has some advantages over the direct mode:
-1. The flows' are buffered in the Kafka topic, so if there is a peak of flows, we make sure that
-   FLP will receive/process them without any kind of denial of service.
-2. Flows are persisted in the topic, so if FLP is restarted by any reason (an update in the
-   configuration or just a crash), the forwarded flows are persisted in Kafka for its later
-   processing, and we don't lose them.
-3. Deploying FLP as a deployment, you don't have to keep the 1:1 proportion. You can scale up and
-   down FLP pods according to your load.
\ No newline at end of file
+# NetObserv architecture
+
+_See also: [architecture in the downstream documentation](https://docs.openshift.com/container-platform/latest/observability/network_observability/understanding-network-observability-operator.html#network-observability-architecture_nw-network-observability-operator)_
+
+NetObserv is a collection of components that can sometimes run independently, or as a whole.
+
+The components are:
+
+- An [eBPF agent](https://github.com/netobserv/netobserv-ebpf-agent), that generates network flows from captured packets.
+  - It is attached to any/all of the network interfaces in the host, and listens for packets (ingress+egress) with [eBPF](https://ebpf.io/).
+  - Packets are aggregated into logical flows (similar to NetFlows), periodically exported to a collector, generally FLP.
```

Comment:
Should we mention the packet capture here ?
That may come in the operator too in the future